pages/storage_and_backup/object_storage/s3_asynchronous_replication/guide.en-gb.md

```diff
@@ -0,0 +1,378 @@
+|--------------------------------------------------------------|--------------------------------------------------------------|
+| Objects created AFTER the upload of the replication configuration | Delete marker i.e., objects deleted in the source bucket are not automatically deleted in the destination bucket |
+| Unencrypted objects                                          | Object replicas i.e., objects that are the result of a previous replication operation |
+| Objects encrypted with customer provided keys (SSE-C)        | Objects that have already been replicated to a previous destination |
+| Object metadata from the source objects to the replicas      | Objects that are stored in the Cold Archive temporary storage |
+| Objects in the source bucket the bucket owner has permissions to read and access ACLs | Bucket configurations i.e., lifecycle configuration, CORS configuration, bucket ACLs, etc. |
+| Object ACL updates                                           | Actions resulting from Lifecycle Configuration actions |
+| Object tags                                                  | Objects created BEFORE the upload of the replication configuration |
+| S3 Object Lock retention configuration                       | Replication to a bucket in a different Public Cloud Project i.e., source and destination buckets must be in the same project |
+
+
+### Replication Configuration
+
+A replication configuration is defined through a set of rules within a JSON document. This document is uploaded and applied to the source bucket, detailing how objects are to be replicated.
+
+### Each Replication Rule Defines:
+
+- A **unique rule ID** to identify the rule.
+- **Rule priority** to determine the order of execution when multiple rules exist.
+- **Destination bucket** where the replicated objects will be stored.
+- **Objects to be replicated**: By default, all objects are eligible for replication. However, you can specify a subset of objects by filtering them with a prefix and/or tags.
+- **Optional target storage class**: By default, object replicas will inherit the same storage class as the source objects. If needed, you can specify a different storage class for the replicas.
+
+### Replication Rule Structure
+
+The basic structure of a replication rule within the configuration JSON document is as follows:
+
+
+```json
+{
+  "Role": "string",
+  "Rules": [
+    {
+      "ID": "string",
+      "Priority": integer,
+      "Filter": {
+        "Prefix": "string",
+        "Tag": {
+          "Key": "string",
+          "Value": "string"
+        },
+        "And": {
+          "Prefix": "string",
+          "Tags": [
+            {
+              "Key": "string",
+              "Value": "string"
+            }
+          ]
+        }
+      },
+      "Status": "Enabled"|"Disabled",
+      "Destination": {
+        "Bucket": "string",
```

Comment:
feedbacks from dev team; replace "string" in "Bucket" attribute by "arn:aws:s3:::<your_bucket_name>"